docs/guide/architecture.md

```diff
@@ -0,0 +1,42 @@
+# Architecture
+
+## General information
+
+The monorepo is powered by two technologies:
```

Comment:
i would just say "The monorepo is powered by :", not sure if technologies is the right word :)